pkg/scheduler_plugins/candidate/plugin.go

```diff
@@ -28,23 +29,26 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/klog/v2"
 	"k8s.io/kubernetes/pkg/scheduler/framework"
-	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	crConfig "sigs.k8s.io/controller-runtime/pkg/client/config"
 
+	"admiralty.io/multicluster-scheduler/pkg/apis/config"
 	"admiralty.io/multicluster-scheduler/pkg/common"
 	"admiralty.io/multicluster-scheduler/pkg/generated/clientset/versioned"
 )
 
 type Plugin struct {
 	handle framework.Handle
 	client versioned.Interface
+
+	preBindWaitDuration time.Duration
 }
 
 var _ framework.PreFilterPlugin = &Plugin{}
 var _ framework.ReservePlugin = &Plugin{}
 var _ framework.PreBindPlugin = &Plugin{}
 
 // Name is the name of the plugin used in the plugin registry and configurations.
-const Name = "candidate"
+const Name = "Candidate"
```

Comment:
The framework expects the args to be of the form `pluginName + "Args"` so I changed the casing here.